src/api/public/apidocs-new/paths/build_project_name_repository_name_architecture_name_package_name_buildinfo.yaml

```diff
@@ -33,3 +33,93 @@ get:
 
   tags:
     - Build
+
+post:
+  summary: Return build information about a specific package.
```

Comment:
```suggestion
  summary: Set the build information about a specific package.
```

Reply:
Nop. This endpoint doesn't set any build information.